README.md

```diff
@@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
 
 Only simple products are synced. Configurable products are not synced.
 
+## Requirements
+
+In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
+declared in the crontab via the following:
+
+php cron.php -mdefault 1
```

Comment:
Yan,

The lines above were given as they should appear.

php /path/to/magento/cron.php -mdefault 1


or


/bin/sh /path/to/magento/cron.sh cron.php -mdefault 1 > /dev/null 2>&1

are the actual lines which should appear in the crontab (only one of
them, not both of them)


On Fri, Aug 21, 2015 at 10:05 AM, Yan Pritzker <notifications@github.com>
wrote:

> In README.md
> <https://github.com/reverbdotcom/magento/pull/3#discussion_r37643190>:
>
> > @@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
> >
> >  Only simple products are synced. Configurable products are not synced.
> >
> > +## Requirements
> > +
> > +In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
> > +declared in the crontab via the following:
> > +
> > +php cron.php -mdefault 1
>
> what is the full format of the line in crontab as it should appear? can we
> give a command to modify the crontab. something like this
>
> (crontab -l ; echo "0 * * * * php /path/to/magento/app/reverb/cron.php") | crontab -
>
> I made up the path as I'm not sure where that would live.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reverbdotcom/magento/pull/3/files#r37643190>.
>